docs/run-node/interact-node.md

```diff
@@ -0,0 +1,49 @@
+<!--
+order: 3
+-->
+
+# Interacting with the Node
+
+## Pre-requisite Readings
+
+- [Running a Node](./run-node.md) {prereq}
+
+## Via CLI
+
+Now that your chain is running, it is time to try sending tokens from the first account you created to a second account. In a new terminal window, start by running the following query command:
+
+```bash
+simd query account $(simd keys show my_validator -a) --chain-id my-test-chain
+```
+
+You should see the current balance of the account you created, equal to the original balance of `stake` you granted it minus the amount you delegated via the `gentx`. Now, create a second account:
+
+```bash
+simd keys add receiver
```

Comment:
```suggestion
simd keys add receiver
export RECEIVER=$(simd keys show receiver -a)
```

Reply:
We don't need to use the `export` statement though. For the sake of clarity and if we assume that the user runs all the commands in one terminal window/session, we can remove `export`.